### DISCUSSION TITLE
Status Monitor Hull Water Detection rounds down to 0% when between 0~0.99%

### DISCUSSION THREAD

**Discussion by Kaillera** [FakeFishGames/Barotrauma#8964] [Ideas and suggestions]
- [ ] I have searched the issue tracker to check if the issue has already been reported.

**Description**
Flood alarms logic tends to go off whenever there's still more than 0%, but less than 1% of water. When a hull with no ducts or pumps are filled with water and are drained via opening doors, the remaining 1% water takes a long time to flush out to exactly 0%. When players are viewing the status monitor trying to deduce what rooms are still flooded, if it's less than 1%, the hulls on the monitor would round down saying 0%, when it's not.

I suggest it should round up instead.

![Capture](https://user-images.githubusercontent.com/5086682/164991919-9a75a4f1-0afa-4f87-b9e3-3cd4bc090d20.PNG)
![Capture](https://user-images.githubusercontent.com/5086682/164991930-313a45c0-f57a-4106-b63b-817003f1c8c0.PNG)

**Steps To Reproduce**
The fastest example I can give is my sub: https://steamcommunity.com/sharedfiles/filedetails/?id=2683155573

Cause a leak in the kitchen, or top gunnery, or engine room.

**Version**
~0.17.13.0

**Comment by James-E-A** [comment]
A ghetto mitigation for this: use the up-arrow on the floors in the sub editor to create the world's shallowest downhill gradient from rooms that don't have vents into rooms that do
